File: Logic_HandsOn/C++/Problems on bit,File Handing,Pattern etc/program119.cpp
///////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
//
//	Program:
//		Write a program which accept string from user and copy that	characters of that string 
//		into another string by converting all capital characters into small case.
//
//////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////



#include<iostream>
using namespace std;

class Solution
{
	private:
		char *src,*dest;

	public:
		Solution(char *src,char *dest)
		{
			this->src=src;
			this->dest=dest;
		}

		void StrCpySmall()
		{

			while(*src != '\0')
			{
				if(*src>='A' && *src<='Z')
				{	
					*dest = (*src)+32;
				}
				else if(*src>='a' && *src<='a')
				{
					*dest=*src;
				}
				else
				{
					*dest = *src;
				}
				dest++;
				src++;
			}
			*dest='\0';

		}

};


int main()
{
	char arr[30] = "Vishal Patil";
	char brr[30];

	Solution *sObj = new Solution(arr,brr);

	sObj->StrCpySmall();
	cout<<brr<<"\n";

	return 0;
}

/*

	OUTPUT:
		vishal patil
*/

File: Logic_HandsOn/Java/Problems on bit,file handling,String etc/Program119.java

///////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
//
//	Program:
//		Write java program which accept N numbers from user and accept Range, Display all elements from that range
//
//////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
import java.util.Scanner;

class ArrayX
{
	public int iSize,Brr[],iEnd,iStart;

	public ArrayX(int iSize,int iStart,int iEnd)
	{
		Brr = new int[iSize];
		this.iStart=iStart;
		this.iEnd=iEnd;
	}

	public void Accept()
	{
		int iCnt=0;
		Scanner sObj = new Scanner(System.in);
		System.out.println("Enter elements in Array: ");
		for(iCnt=0;iCnt<Brr.length;iCnt++)
		{
			Brr[iCnt] = sObj.nextInt();
		}
	}
}

class Number extends ArrayX
{

	public Number(int iValue,int iStart,int iEnd)
	{
		super(iValue,iStart,iEnd);
	}

	public void Display()
	{
		int iCnt=0;
		for(iCnt=0;iCnt<Brr.length;iCnt++)
		{
			if(Brr[iCnt]>=iStart && Brr[iCnt]<=iEnd)
			{
				System.out.print(Brr[iCnt]+" ");
			}
		}

		System.out.println();
	}		
}

class Program4

{
	public static void main(String arg[])
	{
		int iCnt=0;

		Scanner sObj = new Scanner(System.in);
		System.out.println("Enter number of elements: ");
		int iSize = sObj.nextInt();

		System.out.println("Enter the Start: ");
		int iStart = sObj.nextInt();
		
		System.out.println("Enter the End: ");
		int iEnd = sObj.nextInt();

		Number nObj = new Number(iSize,iStart,iEnd);
		nObj.Accept();
		nObj.Display();

	}
}

/*
OUTPUT:
Enter number of elements: 
6
Enter the Start: 
60
Enter the End: 
90
Enter elements in Array: 
85
66
3
76
93
88
85 66 76 88 


*/
